src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -92,9 +128,37 @@ const ScopeSelectionScreen = ({ navigation, route }: IntroScreenProps): JSX.Elem
           )}
           <StyledText>{getLabels().scopeSelection.selectProfession}</StyledText>
         </TextContainer>
-        <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
-          <DisciplineContainer>{disciplineItems}</DisciplineContainer>
-        </ServerResponseHandler>
+        <SearchContainer>
+          <SearchBar
+            query={searchString}
+            setQuery={input => {
+              setSearchString(input)
+              setShowSearchResults(input.length > 0)
+            }}
+            placeholder={getLabels().scopeSelection.searchProfession}
+          />
+        </SearchContainer>
+        {showSearchResults ? (
+          <ScopeContainer>
+            {filteredProfessions?.map(profession => (
+              <Pressable
+                key={profession.id}
+                onPress={async () => {
+                  await pushSelectedProfession(profession.id).then(() => {
+                    navigation.navigate('ManageSelection')
+                  })
+                }}>
+                {highlightText(splitTextBySearchString(profession.title, searchString))}
+                <Text>{profession.parentTitle}</Text>
```

Comment:
Is the parentTitle always null? I never see it in the search.

Reply:
Yes, it is. I decided to put it in, in case someone fills that field in the future.